File: Solution/WebSolution/AssessmentPortal/Microservices/WebAPI/AssessmentIntelligenceAPI/Services/AssessmentIntelligenceService.cs
using Transflower.Assessment.WebAPI.AssessmentIntelligenceAPI.Entities;
using Transflower.Assessment.WebAPI.AssessmentIntelligenceAPI.Repositories.Interfaces;
using Transflower.Assessment.WebAPI.AssessmentIntelligenceAPI.Services.Interfaces;

namespace Transflower.Assessment.WebAPI.AssessmentIntelligenceAPI.Services{
public class AssessmentIntelligenceService:IAssessmentIntelligenceService
{
    private readonly IAssessmentIntelligenceRepository _repo;

    public AssessmentIntelligenceService(IAssessmentIntelligenceRepository repository){
        _repo=repository;
    }

    public async Task<List<AnnualCandidateResult>> GetCandidateResults(int candidateId, int year){
        return await _repo.GetCandidateResults(candidateId, year);
    }
}
}

File: Solution/Java/tflassessment-demo/src/main/java/com/transflower/tflassessment/demo/services/AssessmentIntelligenceService.java
package com.transflower.tflassessment.demo.services;

import java.util.List;

import com.transflower.tflassessment.demo.entities.AnnualCandidateResult;

public interface AssessmentIntelligenceService {
    List <AnnualCandidateResult> getCandidateResults(int candidateId, int year);
}
